elements/bulbs-video/components/revealed.js

```diff
@@ -206,6 +206,7 @@ export default class Revealed extends React.Component {
       flashplayer: '//ssl.p.jwpcdn.com/player/v/7.4.3/jwplayer.flash.swf',
       aspectratio: '16:9',
       autostart: true,
+      hlshtml: true,
```

Comment:
@collin, we have to omit that until we are HTML5 certified by an ad tech vendor for one of our live campaigns.  JWPlayer should have a bug fix for that one issue in their 7.5 beta release that's ready this week or next.